src/mapgen_functions.cpp

```diff
@@ -4608,3 +4608,22 @@ void place_stairs( mapgendata &dat )
         }
     }
 }
+
+void resolve_regional_terrain_and_furniture( const mapgendata &dat )
+{
+    for( int i = 0; i < SEEX * 2; i++ ) {
+        for( int j = 0; j < SEEY * 2; j++ ) {
```

Comment:
```suggestion
        for( const tripoint &p : dat.m.points_on_zlevel() ) {
```
That way you only have one loop, and it does not use magic numbers and it will be correct even if the mapgen map has a different size in the future.

Reply:
Great, thanks for the reminder. This was a change I wrote months ago before that existed and just rebased, so I'd completely forgotten about that helper you added. :+1: